src/assets/scripts/client/aircraft/FlightManagementSystem/WaypointModel.js

```diff
@@ -188,6 +188,33 @@ export default class WaypointModel {
     }
 
     /**
+     * @for WaypointModel
+     * @property hasAltitudeRestriction
+     * @type {boolean}
+     */
+    get hasAltitudeRestriction() {
+        return this.altitudeMaximum !== -1 || this.altitudeMinimum !== -1;
+    }
+
+    /**
+     * @for WaypointModel
+     * @property hasRestriction
+     * @type {boolean}
+     */
+    get hasRestriction() {
```

Comment:
Well they kinda just seemed more appropriate as getters. I guess I've been thinking about it as if you are trying to retrieve something rather than do something, I've been tending to make that thing a getter.

And as for the order, really we just need some way to keep things organized. Enormous classes with 50+ methods I'm finding _SO_ difficult to navigate. The simplest and most obvious solution I think is to alphabetize them. If not, they could be grouped by relevance or something, but there should be like comments dividing up the sections, titling the below methods, so you know what realm you're looking through, etc... it all probably sounds like much ado about nothing, but I legitimatly struggle to find what I'm looking for whenever I look through the lists of properties and methods, and would like to start doing something to lessen that as much as we can.